### ISSUE TITLE
Missing html docs for miscellaneous_ex14

### ISSUE THREAD

**Issue by jwpeterson** [libMesh/libmesh#3087]
Hi @BalticPinguin,

Since you originally added this example, and it uses both complex numbers and infinite elements (a build I don't have readily available) would you mind generating the html documentation for it by doing the following (assuming your libmesh checkout is located at `$LIBMESH_ROOT` and your build directory is `$LIBMESH_ROOT/build`):
```
cd $LIBMESH_ROOT/build/examples/miscellaneous/miscellaneous_ex14
make doc
cp miscellaneous_ex14.html $LIBMESH_ROOT/doc/html/examples
```
and then creating a libMesh PR with the new file (or just sending it to me directly if you prefer)? 

Thanks!

**Comment by BalticPinguin** [comment]
sorry for the delay: Sure I will do this.

**Comment by jwpeterson** [comment]
Fixed by #3090
